packages/components/src/internal/components/auditlog/constants.ts

```diff
@@ -52,7 +52,7 @@ export const USER_AUDIT_QUERY: AuditQuery = {
     label: 'User Events',
     value: 'userauditevent',
 };
-export const ASSAY_AUDIT_QUERY: AuditQuery = { value: 'experimentauditevent', label: 'Assay Events' };
+export const ASSAY_AUDIT_QUERY: AuditQuery = { value: 'assayauditevent', label: 'Assay Events' };
```

Comment:
was this wrong before? or is this audit query name changing with this PR?

Reply:
It was not wrong before. The old query no longer work for apps because it relied on LSID (using q~) containing the assay design name to filter down to the assay design. With lsid no longer containing name, I had to query for protocol name for it to be available. Test cases updated in test plan to cover this.